packages/cra-to-nx/src/index.ts

```diff
@@ -27,6 +27,11 @@ export const commandsObject = yargs
     describe: 'Use Vite and Vitest (instead of Webpack and Jest)',
     default: false,
   })
+  .option('nested', {
```

Comment:
Let's rename this `integrated` and default to `false` so projects are converted to nested project (smaller changeset) rather than integrated monorepo.